doc/index.rst

```diff
@@ -2,78 +2,31 @@ metric-learn: Metric Learning in Python
 =======================================
 |License| |PyPI version|
 
-Distance metrics are widely used in the machine learning literature.
-Traditionally, practicioners would choose a standard distance metric
-(Euclidean, City-Block, Cosine, etc.) using a priori knowledge of
-the domain.
-Distance metric learning (or simply, metric learning) is the sub-field of
-machine learning dedicated to automatically constructing optimal distance
-metrics.
-
-This package contains efficient Python implementations of several popular
-metric learning algorithms.
+Welcome to metric-learn's documentation !
```

Comment:
this appears with same title level as the main title (metric-learn: Metric Learning in Python)
better to use lower level / smaller text size

Reply:
That's right, done